src/UI/Pages/CompileReports/FilterWidget/StatusFilter.tsx

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import { ToolbarFilter } from "@patternfly/react-core";
+import { CompileReportParams, CompileStatus } from "@/Core";
+import { SingleTextSelect } from "@/UI/Components";
+import { words } from "@/UI/words";
+
+interface Props {
+  isVisible: boolean;
+  selected: string | null;
+  setSelected: (selected: string | null) => void;
+}
+
+export const StatusFilter: React.FC<Props> = ({
+  isVisible,
+  selected,
+  setSelected,
+}) => {
+  const compileStatuses = Object.keys(CompileStatus).map(
```

Comment:
You could also have a namespace `CompileReports`.
And then You can have `CompileReports.Status` & `CompileReports.StatusList`.
But not really required.